File: mobile/flutter/flutter_plugins/flutter_codesdancing/ios/Classes/CodesDancingPlugin.m
#import "CodesDancingPlugin.h"
#import "FlutterBoostPlugin.h"

@interface CodesDancingPlugin()
@end

@implementation CodesDancingPlugin

+ (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar>*)registrar {
    [FlutterBoostPlugin registerWithRegistrar:registrar];
}
@end

File: mobile/flutter/flutter_plugins/flutter_codesdancing/android/src/main/java/com/codesdancing/flutter/CodesDancingPlugin.java
package com.codesdancing.flutter;

import com.idlefish.flutterboost.FlutterBoostPlugin;

import io.flutter.plugin.common.PluginRegistry;

public class CodesDancingPlugin {
    public static void registerWith(PluginRegistry.Registrar registrar) {
        FlutterBoostPlugin.registerWith(registrar);
    }
}
